series/brno-pyvo/events/2018-11-29.yaml

```diff
@@ -11,14 +11,23 @@ talks:
         Nastavení rozumného vývojového prostředí a správa softwaru ve Windows nemusí být nejpříjemnější. Ukážu "balíčkovací systém" chocolatey, ipython, msys2 a rozumnou příkazovou řádku s příkazy, na které jsem zvyklý z Linuxu.
     speakers:
       - Tomáš Mikulec
+    coverage:
+      - video: https://www.youtube.com/watch?v=VyEoUFrulJs
   - title: Projekty, aneb Kafe s Pomerančem
     description: |
         Během svých Python kurzů mě napadá spousta projektů, v cestě ale stojí jednak jakým způsobem zaujmout účastníky a ukázat jim, co vše je možné, a jednak jak jim po pochopení základů dát jednodušší projekty, do kterých se můžou hned pustit.
         Kam všechny nápady sepsat tak, aby si mohli nováčci dobře vybrat? Bude to zajímavé i pro ostatní? Asi mám nápad na řešení.
     speakers:
       - Tomáš Mikulec
-  - title: "Lightning talky"
-    description: |
-        Dojde určitě i na lightning talky. Jeden můžeš mít i ty.
-        Viz [pyvo.cz#info-for-speakers](https://pyvo.cz/#info-for-speakers).
-        Registrace není nutná.
+    coverage:
+      - video: https://www.youtube.com/watch?v=tdQMFW10WTc
+  - title: Docs with Sphinx
+    speakers:
+      - Martin Javorek
```

Comment:
@encukou Na videu je napsaný Martin, ale jsem přesvědčený, že to mluví Honza. Jde to poznat podle toho mávání rukama.

Reply:
#178